TASK/TaskPlaneTaxi.md

```diff
@@ -6,15 +6,17 @@ aliases: ["0x92C360B5F15D2302"]
 
 ```c
 // 0x92C360B5F15D2302
-void _TASK_PLANE_TAXI(Ped pilot, Vehicle aircraft, Any p2, Any p3, Any p4, Any p5, Any p6);
+void _TASK_PLANE_TAXI(Ped pilot, Vehicle aircraft, cs_type(Any) Number xPos, cs_type(Any) Number yPos, cs_type(Any) Number zPos, cs_type(Any) Number speed, Any p6);
```

Comment:
`Number` isn't a valid type, you should use C primitive types

Reply:
I believe it should be good now